topics/admin/tutorials/ansible-galaxy/tutorial.md

```diff
@@ -2341,8 +2341,8 @@ With Ansible, upgrading Galaxy to a new release is incredibly easy. Here is a co
 @@ -345,7 +345,7 @@ galaxy_instance_hostname: usegalaxy.eu
  
  galaxy_repo: 'https://github.com/usegalaxy-eu/galaxy.git'
--galaxy_commit_id: 'release_19.05'
-+galaxy_commit_id: 'release_19.09'
+-galaxy_commit_id: 'release_20.09'
```

Comment:
Not sure it's important to change these, it was from a specific commit, other upgrades could have more changes, etc